roaring/roaring.go

```diff
@@ -93,6 +93,7 @@ type Containers interface {
 	// return the first container at or after key. found will be true if a
 	// container is found at key.
 	Iterator(key uint64) (citer ContainerIterator, found bool)
+	Count() uint64
```

Comment:
At first I was a little confused by the name of this method; "does this count the number of containers?". But I suppose as long as the comment specifies that `Count returns the number of bits set in the containers.` then it's ok.

Reply:
I've been trying to figure out which fields we made available for the b+tree stuff.
It looks like that's in commit 4282e90f where we added a `Mapped()` getter method and an `Update()` method. Would adding a `N()` getter method work for you?

Reply:
Works for me, like it @codysoyland ?

Reply:
Seems worth the export 👍